README.md

```diff
@@ -1,2 +1,94 @@
-# triton-touchbase
-Touchbase stack designed for container-native deployment on Joyent's Triton platform
+triton-touchbase
+==========
+
+*[Touchbase](https://github.com/couchbaselabs/touchbase) stack designed for container-native deployment on Joyent's Triton platform*
+
+This repo is a demonstation of a multi-tier container-native application architecture on [Joyent's](https://www.joyent.com/) Triton platform. It uses Touchbase and its supporting services as an example. It includes the following components:
+
+- Consul, acting as a discovery service
+- Couchbase, for the data tier
+- Touchbase, a Node.js application
+- Nginx, acting as a load balancer for Touchbase nodes
+- CloudFlare-watcher, to update DNS
+- CloudFlare DNS, to make the site accessible by domain name on the Internet
+
+![Diagram of Touchbase architecture](./doc/triton-touchbase.png)
+
+### Couchbase
```

Comment:
I'm a bit tempted to remove the list of components as a way to minimize the duplication of content between this and the blog post.